tests/attacks/evasion/test_shadow_attack.py

```diff
@@ -34,18 +34,13 @@ def fix_get_mnist_subset(get_mnist_dataset):
     (x_train_mnist, y_train_mnist), (x_test_mnist, y_test_mnist) = get_mnist_dataset
     n_train = 100
     n_test = 11
-    yield (x_train_mnist[:n_train], y_train_mnist[:n_train], x_test_mnist[:n_test], y_test_mnist[:n_test])
+    yield x_train_mnist[:n_train], y_train_mnist[:n_train], x_test_mnist[:n_test], y_test_mnist[:n_test]
 
 
-@pytest.mark.only_with_platform("pytorch")
```

Comment:
This test is using the framework independent fixture `get_image_classifier_list_for_attack` which is great. However based on the previous comment in `run_test.sh` this marker might need to be added if this test currently does not work for any framework other than pytorch `@pytest.mark.skipMlFramework("tensorflow", "keras", "scikitlearn")` Ideally, there should be no need for this marker since the test should work for all frameworks but in the event that it does not then this marker should be used withe the appropriate frameworks.

Reply:
I have added `@pytest.mark.skipMlFramework`.